### DISCUSSION TITLE
[공지사항] 멘티님들~~미비된 과제 독촉합니다~~

### DISCUSSION THREAD

**Discussion by yoomambo** [4Moyede/HexatonClass-01#127] [00_공지사항]
## 지우

- 2주차 과제리뷰보고 파일 다시 수정해서 재업로드하기
- 2주차 멘티들 코드리뷰 해서 discussion에서 2주차 과제리뷰에 댓글달기
- 2주차 이슈 리뷰 정리해서 마크다운파일에 저장 후 업로드
- 1주차 과제 3번문제 수정안했으니 피드백 받아서 수정한 후 재업로드

## 성준

- 2주차 과제리뷰보고 파일 다시 수정해서 재업로드하기
- 2주차 멘티들 코드리뷰 해서 discussion에서 2주차 과제리뷰에 댓글달기
- 2주차 이슈 리뷰 정리해서 마크다운파일에 저장 후 업로드

## 경훈

- 2주차 과제리뷰보고 파일 다시 수정해서 재업로드하기
- 2주차 멘티들 코드리뷰 해서 discussion에서 2주차 과제리뷰에 댓글달기
- 2주차 이슈 리뷰 정리해서 마크다운파일에 저장 후 업로드
- 03폴더 02로 이름 수정하기

## 덕규

- 미비된 과제는 아니지만, 덕규멘티님 파일이 pandas homework 로 되어있는데 띄어쓰기는 지양하시고 _ 언더바로 바꿔서 다시올리시길 바랍니다.

## 재범

- 2주차 멘티들 코드리뷰 해서 discussion에서 2주차 과제리뷰에 댓글달기
- 2주차 이슈 리뷰 정리해서 마크다운파일에 저장 후 업로드

**Comment by yoomambo** [comment]
@lsj0822 @NA-DK @dlrudgnsdlek @jiwoo4424 언제쯤 제출하시는지요..

**Reply by lsj0822** [reply]
> @lsj0822 @NA-DK @dlrudgnsdlek @jiwoo4424 언제쯤 제출하시는지요..

@yoomambo 지금 코드 리뷰하고 있습니다! 최대한 빠르게 제출하겠습니다...

**Reply by dlrudgnsdlek** [reply]
인터넷 연결 문제로 주말에 못했습니다ㅠㅠ 늦어도 내일 오전까지 꼭 올리겠습니다! 죄송합니다ㅠ

**Reply by jiwoo4424** [reply]
> @lsj0822 @NA-DK @dlrudgnsdlek @jiwoo4424 언제쯤 제출하시는지요..

죄송해요 계속 일이 너무 많았어서 ... 오늘 안에 최대한 올리도록 하겠습니다
